### PR TITLE
docs: add Celestia-ContributorGitPOAP sections

### PR DESCRIPTION
**Pull request by SakiAoiaoi**
Add Celestia Contributor GitPOPA sections to CONTRIBUTING
Close https://github.com/celestiaorg/celestia-core/issues/1155

- [ ✅] Tests written/updated
- [ ✅] Changelog entry added in `.changelog` (we use
  [unclog](https://github.com/informalsystems/unclog) to manage our changelog)
- [ ✅] Updated relevant documentation (`docs/` or `spec/`) and code comments

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -38,6 +38,9 @@ coherence in the larger context. If you are not comfortable with writing an ADR,
 you can open a less-formal issue and the maintainers will help you turn it into
 an ADR.
 
+Once your pull requests are merged to one of celestiaorg's GitHub repositories, you will be automatically awared  the right to mint a [Celestia Contributor GitPOAP](https://www.gitpoap.io/gh/celestiaorg/celestia-core). (You can check which repositories here: [Celestia Contributor GitPOAP](https://www.gitpoap.io/gh/celestiaorg/celestia-core))
```
Comment (rootulp) [inline CONTRIBUTING.md:41]:
```suggestion
Once your pull request is merged to an eligible celestiaorg GitHub repository, you will be rewarded the ability to mint a [GitPOAP](https://www.gitpoap.io). You can check which repositories are eligible at [gitpoap/celestiaorg](https://www.gitpoap.io/gh/celestiaorg)
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -38,6 +38,9 @@ coherence in the larger context. If you are not comfortable with writing an ADR,
...
+
```
Comment (rootulp) [inline CONTRIBUTING.md:42]: Too many new lines

```suggestion
```

**Review by rootulp — CHANGES_REQUESTED** [review]
I like the idea but I I'd rather we don't modify this file because this repo is a fork of upstream CometBFT and AFAIK there aren't any Celestia specific things in CONTRIBUTING.md.

If we minimize the diff, it's easier to pull changes from upstream.

File: CONTRIBUTING.md (hunk 1 above)
Comment (SakiAoiaoi) [reply CONTRIBUTING.md:41]: Thanks a lots!Your kind points motivate me so much.

File: CONTRIBUTING.md (hunk 2 above)
Comment (SakiAoiaoi) [reply CONTRIBUTING.md:42]: Thanks a lot! Your kind points motivate me so much.

**Comment by SakiAoiaoi** [comment]
I completely agree with you!!Thank you for your kind correcting me.

I think it would be better to add the GitPOAP explanation to the README, so I made a new pull request to the README to it.
https://github.com/celestiaorg/celestia-core/pull/1157

**Pull request closed without merging**